app/src/common/services/notification/notificationService.ts

```diff
@@ -33,11 +40,31 @@ namespace common.services.notification {
         }
 
         /**
+         * Add a delay before showing the toast
+         *
+         * @param milliseconds
+         */
+        public delay(milliseconds:number) {
+
+            this.timeOut = milliseconds;
+
+            return this;
+
+        }
+
+        /**
          * Show the toast
          */
         public pop():void {
 
-            this.$mdToast.show(this.toastOptions);
+            if(this.timeOut > 0) {
```

Comment:
With the above, this should just be a check for `_.isNumber(this.timeout)` as 0 is valid.